class.photon.php

```diff
@@ -60,6 +60,9 @@ private function setup() {
 
 		// Responsive image srcset substitution
 		add_filter( 'wp_calculate_image_srcset', array( $this, 'filter_srcset_array' ), 10, 4 );
+		if ( ! has_filter( 'wp_calculate_image_sizes') ) { // To allow themes to fully control the sizes array.
```

Comment:
Could we achieve the same effect by setting a lower priority to our filter, since if a theme had defined their own, it would be run and filtered later (assuming default priority of 10)?